homeassistant/components/aftership/config_flow.py

```diff
@@ -0,0 +1,97 @@
+"""Config flow for aftership integration."""
+import logging
+
+from pyaftership.tracker import Tracking
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_API_KEY, CONF_NAME, HTTP_OK
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME, default="My AfterShip Packages"): str,
+        vol.Required(CONF_API_KEY, default=""): str,
+    }
+)
+
+
+class AfterShipTest:
+    """Placeholder class to make tests pass."""
+
+    def __init__(self, hass, session, api_key):
+        """Initialize."""
+        self.api_key = api_key
+        self.hass = hass
+        self.session = session
+
+    async def authenticate(self) -> bool:
+        """Test if we can authenticate with the host."""
+        aftership = Tracking(self.hass.loop, self.session, self.api_key)
+
+        await aftership.get_trackings()
+
+        if not aftership.meta or aftership.meta["code"] != HTTP_OK:
+            _LOGGER.error(
+                "No tracking data found. Check API key is correct: %s", aftership.meta
+            )
+            return
```

Comment:
```suggestion
            return False
```
This method is supposed to return a boolean.